### PR TITLE
Missing use iis plat handler in the sample

### PR DESCRIPTION
**Pull request by Kagamine**

### CODE REVIEW

File: samples/IISSample/Startup.cs [hunk 1]
```diff
@@ -15,6 +15,8 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
 
             app.Run(async (context) =>
             {
+                app.UseIISPlatformHandler();
```
Comment (Tratcher) [inline samples/IISSample/Startup.cs]: Hmm, how did this get forgotten... Note this should be on line 15 outside the Run.

**Comment by davidfowl** [comment]
:shipit:

File: samples/IISSample/Startup.cs (hunk 1 above)
Comment (Kagamine) [inline samples/IISSample/Startup.cs]: :)

**Comment by Kagamine** [comment]
I just made a PR on Mvc https://github.com/aspnet/Mvc/pull/3224
Refer to this sample, I missed the use iis plat handler also.

**Comment by Tratcher** [comment]
Note you don't actually need the middleware or package dependency in common scenarios, it only deals with header forwarders. All the magic to run in IIS happens in the web.config generated by dnu publish.

**Comment by Kagamine** [comment]
:smile:  thanks

**Comment by Kagamine** [comment]
Done.

**Comment by Tratcher** [comment]
Rebased and merged.

**Pull request closed without merging**